src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -302,6 +304,18 @@ class WholeBodyDynamicsDevice :  public yarp::dev::DeviceDriver,
      */
     bool validOffsetAvailable;
 
+    /**
+     * Double to keep track of last time skin was read
+     * (to verify timeouts).
+     */
+    double  last_reading_skin_contact_list_Stamp;
```

Comment:
Nitpick, but given that all attributes are using CamelCase naming, I would suggest to use CamelCase (with lower capital letter) also for this parameter.